clients/filesystem-hadoop3-runtime/build.gradle.kts

```diff
@@ -43,6 +43,8 @@ tasks.withType<ShadowJar>(ShadowJar::class.java) {
   relocate("com.github.benmanes.caffeine", "org.apache.gravitino.shaded.com.github.benmanes.caffeine")
   // relocate common lang3 package
   relocate("org.apache.commons.lang3", "org.apache.gravitino.shaded.org.apache.commons.lang3")
+  relocate("org.apache.hc", "org.apache.gravitino.shaded.org.apache.hc")
+  relocate("org.checkerframework", "org.apache.gravitino.shaded.org.checkerframework")
```

Comment:
How about other dependencies here?

```  relocate("org.apache.commons", "org.apache.gravitino.shaded.org.apache.commons")
  relocate("org.apache.hc", "org.apache.gravitino.shaded.org.apache.hc")
  relocate("org.apache.http", "org.apache.gravitino.shaded.org.apache.http")
  relocate("org.apache.thrift", "org.apache.gravitino.shaded.org.apache.thrift")
  relocate("org.checkerframework", "org.apache.gravitino.shaded.org.checkerframework")
 ```

Reply:
@mchades @jerryshao ^^

Reply:
The dependencies of `org.apache.http` and `org.apache.thrift` already shaded by the `client-java-runtime`